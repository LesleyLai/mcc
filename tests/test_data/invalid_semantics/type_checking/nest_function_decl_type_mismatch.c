int main(void)
{
  int foo(void);

  return foo();
}

int foo(int x)
{
  return x + 42;
}
