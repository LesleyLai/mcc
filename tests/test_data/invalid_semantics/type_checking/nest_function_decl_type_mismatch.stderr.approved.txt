{{filename}}:6:5: Error: conflicting types for 'foo'
6 | int foo(int x)
7 | {
8 |   return x + 42;
9 | }

